makefile

```diff
@@ -151,11 +151,7 @@ include make/command
 
 CMDSTAN_VERSION := 2.34.1
 
-ifeq ($(OS),Windows_NT)
-HELP_MAKE=mingw32-make
```

Comment:
You could even go one step further now and just ditch the `help-make` variable entirely and use `MAKE`, I think?

Or just hard code it into the message